app/domain/sac_imports/person_entry.rb

```diff
@@ -52,83 +41,100 @@ def import!
-      mobile = row.fetch(:phone_mobile)
-      direct = row.fetch(:phone_direct)
+    def gender
+      GENDERS[row[:gender]&.to_sym]
+    end
 
-      # reset phone numbers first since import might be run multiple times
-      person.phone_numbers = []
+    def person_type
+      PERSON_TYPES[row[:person_type]&.to_sym] || "person"
+    end
 
-      person.phone_numbers.build(number: phone, label: "Privat") if phone_valid?(phone)
-      person.phone_numbers.build(number: mobile, label: "Mobil") if phone_valid?(mobile)
-      person.phone_numbers.build(number: direct, label: "Direkt") if phone_valid?(direct)
+    def language
+      LANGUAGES[row[:person_type]&.to_sym] || DEFAULT_LANGUAGE
     end
 
-    def build_role(person)
-      return if person.roles.exists?
+    def email
+      row[:email]
+    end
 
-      person.roles.build(
-        group: @group,
-        type: TARGET_ROLE
-      )
+    def company?
+      @is_company ||= person_type == "company"
     end
 
     def parse_address
-      address = row.fetch(:address)
+      address = row[:address]
       return if address.blank?
 
       Address::Parser.new(address).parse
     end
 
+    def assign_attributes(person) # rubocop:disable Metrics/AbcSize
+      person.primary_group = group
+      person.first_name = row[:first_name] unless company?
+      person.last_name = row[:last_name] unless company?
+      person.address_care_of = row[:address_care_of]
+      person.postbox = row[:postbox]
+      person.address = row[:address]
+      person.street = row[:street]
+      person.housenumber = row[:housenumber]
+      person.country = country
+      person.town = row[:town]
+      person.zip_code = row[:zip_code]
+      person.birthday = row[:birthday]
+      person.gender = gender unless company?
+      person.language = language
+      person.family_key = row[:family]
+      person.sac_remark_section_1 = row[:sac_remark_section_1]
+      person.sac_remark_section_2 = row[:sac_remark_section_2]
+      person.sac_remark_section_3 = row[:sac_remark_section_3]
+      person.sac_remark_section_4 = row[:sac_remark_section_4]
+      person.sac_remark_section_5 = row[:sac_remark_section_5]
+      person.sac_remark_national_office = row[:sac_remark_national_office]
+      person.company = company?
+      person.company_name = row[:last_name] if company?
+
+      person.street, person.housenumber = parse_address
+
+      if email.present?
+        person.email = email
+        # Do not call person#confirm here as it persists the record.
+        # Instead we set confirmed_at manually.
+        person.confirmed_at = Time.zone.at(0)
+      end
+    end
+
     def phone_valid?(number)
       number.present? && Phonelib.valid?(number)
     end
 
-    def parse_datetime(value, default: nil)
-      DateTime.parse(value.to_s)
-    rescue Date::Error
-      default
+    def build_phone_numbers(person)
+      # rubocop:disable Lint/SymbolConversion
+      phone_numbers = {
```

Comment:
The first iteration contained multiple phone numbers. With the last version of NAV1 it was merged into one field. I think it doesn't hurt to leave it like this, in case it changes again in the future.